CHANGELOG.md

```diff
@@ -1035,6 +1036,8 @@
 [7176]: https://github.com/enso-org/enso/pull/7176
 [7193]: https://github.com/enso-org/enso/pull/7193
 [7291]: https://github.com/enso-org/enso/pull/7291
+[7176]: https://github.com/enso-org/enso/pull/7176
```

Comment:
There is a `7176` entry just 2 lines above, so this seems like a duplicate.

Reply:
Removed in 90a2c7e50f3ff6095cf2c873ba250316872406f8